tests/atoms/gaia_apps.js

```diff
@@ -296,8 +297,11 @@ var GaiaApps = {
             function() {
               // wait for the displayed app to have expected manifestURL/origin
               let displayedApp = GaiaApps.getDisplayedApp();
+              console.log('*****n****'+displayedApp.entryPoint);
+              console.log('*******'+displayedApp.manifestURL + '::::' + manifestURL);
+              console.log('*******'+displayedApp.entryPoint + '::::' + entryPoint);
               if (displayedApp.manifestURL) {
```

Comment:
I'd add another check here:
```js
if (displayedApp.manifestURL && displayedApp.manifestURL) {
    return displayedApp.manifestURL == manifestURL && displayedApp.entryPoint == entryPoint;
} else if (displayedApp.manifestURL) {
    return displayedApp.manifestURL == manifestURL
} else {
```
because some apps don't have entry points.